src/widgets/barchart/mod.rs

```diff
@@ -8,14 +9,16 @@ pub use bar_group::BarGroup;
 
 use super::{Block, Widget};
 
-/// Display multiple bars in a single widgets
+/// Display multiple [bars](Bar) in a single widget.
```

Comment:
Here do you think it would be good to add a small amount of narrative information about the chart, groups, and bars?

Reply:
Yep, I'll do.

A VHS of the example could be a good fit here. Maybe it's worth opening some sort of RFC or a discussion about it?

Reply:
I was thinking that at some point it might be nice to make each example have a mode that displays just a small render of each feature that can be referenced from the docs easily.